packages/common/src/Constants.ts

```diff
@@ -40,3 +40,8 @@ export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
 // Block number of first EMP created.
 // https://etherscan.io/tx/0x741ccbf0f9655b0b71e3842d788d58770bd3eb80c8f5bdf4fdec7cd74a776ea3
 export const UMA_FIRST_EMP_BLOCK = 10103723;
+
```

Comment:
Do we want this to be in common? Will any other parts our infra need access to this? If not what do you think about placing it within the bots package?

Reply:
Yeah its pretty niche i'll move it to the bots package.